src/services/user-service.ts

```diff
@@ -57,9 +59,50 @@ export class UserService {
     const users = await prisma.user.findMany();
 
     if (users.length === 0) {
-      throw new Error('Nenhum usuário encontrado');
+      throw new GraphQLError('INTERNAL_SERVER_ERROR: No users Where Found', {
+        extensions: {
+          code: '500',
+          additionalInfo: 'Please try again later or create a new user on the database',
+        },
+      });
     }
 
     return users;
   }
+
+  async logInUserService(params: UserLogin): Promise<Login> {
+    const { email, password, rememberMe } = params;
+
+    const user = await prisma.user.findFirst({
+      where: {
+        email,
+      },
+    });
+
+    if (!user) {
+      throw new GraphQLError('USER_NOT_FOUND: cannot find any user with that email', {
+        extensions: {
+          code: '400',
+          additionalInfo: 'Please try again using a different email',
```

Comment:
Haha me da a ideia de ficar chutando emails dessa forma, que tal:
```suggestion
          additionalInfo: 'Please verify email or password and try again',
```

Reply:
Faz sentido, pelo visto as mensagens estavam descritivas até demais  😅